Client/game_sa/CPedSA.h

```diff
@@ -278,134 +308,132 @@ class CPedSA : public virtual CPed, public virtual CPhysicalSA
 
-    BYTE GetOccupiedSeat() { return m_ucOccupiedSeat; }
-    void SetOccupiedSeat(BYTE seat) { m_ucOccupiedSeat = seat; }
+    std::uint8_t GetOccupiedSeat() const noexcept { return m_ucOccupiedSeat; }
+    void SetOccupiedSeat(std::uint8_t seat) noexcept { m_ucOccupiedSeat = seat; }
 
-    void RemoveBodyPart(int i, char c);
+    void RemoveBodyPart(std::uint8_t boneID, std::uint8_t direction);
 
-    void         SetFootBlood(unsigned int uiFootBlood);
-    unsigned int GetFootBlood();
+    void         SetFootBlood(std::uint32_t footBlood);
+    std::uint32_t GetFootBlood();
 
-    bool IsBleeding();
-    void SetBleeding(bool bBleeding);
+    bool IsBleeding() { return GetPedInterface()->pedFlags.bPedIsBleeding; }
+    void SetBleeding(bool bleeding) { GetPedInterface()->pedFlags.bPedIsBleeding = bleeding; }
 
-    bool IsOnFire();
-    void SetOnFire(bool bOnFire);
+    bool IsOnFire() { return GetPedInterface()->pFireOnPed != nullptr; }
+    void SetOnFire(bool onFire);
 
     bool GetStayInSamePlace() { return GetPedInterface()->pedFlags.bStayInSamePlace; }
-    void SetStayInSamePlace(bool bStay);
-
-    void GetVoice(short* psVoiceType, short* psVoiceID);
-    void GetVoice(const char** pszVoiceType, const char** pszVoice);
-    void SetVoice(short sVoiceType, short sVoiceID);
-    void SetVoice(const char* szVoiceType, const char* szVoice);
-    void ResetVoice();
-    void SetLanding(bool bIsLanding) { GetPedInterface()->pedFlags.bIsLanding = bIsLanding; }
+    void SetStayInSamePlace(bool stay) { GetPedInterface()->pedFlags.bStayInSamePlace = stay; }
+
+    void GetVoice(std::int16_t* voiceType, std::int16_t* voiceID);
+    void GetVoice(const char** voiceType, const char** voice);
+    void SetVoice(std::int16_t voiceType, std::int16_t voiceID);
+    void SetVoice(const char* voiceType, const char* voice);
+    void ResetVoice() { SetVoice(m_sDefaultVoiceType, m_sDefaultVoiceID); }
+    void SetLanding(bool isLanding) { GetPedInterface()->pedFlags.bIsLanding = isLanding; }
     void SetUpdateMetricsRequired(bool required) { GetPedInterface()->pedFlags.bUpdateMatricesRequired = required; }
 
     CWeaponStat* GetCurrentWeaponStat();
     float        GetCurrentWeaponRange();
     void         AddWeaponAudioEvent(EPedWeaponAudioEventType audioEventType);
 
-    virtual int GetCustomMoveAnim();
+    virtual int GetCustomMoveAnim() { return m_iCustomMoveAnim; }
     bool        IsDoingGangDriveby();
 
-    CPedIKSAInterface*      GetPedIKInterface() { return &reinterpret_cast<CPedSAInterface*>(m_pInterface)->pedIK; }
-    void*                   GetPedNodeInterface(std::int32_t nodeId) { return reinterpret_cast<CPedSAInterface*>(m_pInterface)->pedNodes[nodeId]; }
+    CPedIKSAInterface*      GetPedIKInterface() { return &GetPedInterface()->pedIK; }
+    void*                   GetPedNodeInterface(std::int32_t nodeId) { return GetPedInterface()->pedNodes[nodeId]; }
```

Comment:
`const` for getters.
`noexcept` if you dont call assembly/other throwing functions.